src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -2053,3 +2053,100 @@ def cli(self, output=None):
                 wireless_info_obj["base_mac"].update(ap_info_dict)
 
         return wireless_info_obj
+
+
+# ======================================
+# Schema for:
+#  * 'show wireless fabric vnid mapping'
+# ======================================
+class ShowWirelessFabricVnidMappingSchema(MetaParser):
+    """Schema for show wireless fabric vnid mapping."""
+
+    schema = {
+      Optional("fabric_vnid_mapping") : {
+        Optional("name"): {
+          Optional(str) : {
+            Optional("l2_vnid"): int,
+            Optional("l3_vnid"): int,
+            Optional("ip_address"): str,
+            Optional("subnet"): str,
+            Optional("control_plane_name"): str
+          }
+        }
+      }
+    }
+
+
+# ======================================
+# Parser for:
+#  * 'show wireless fabric vnid mapping'
+# ======================================
+class ShowWirelessFabricVnidMapping(ShowWirelessFabricVnidMappingSchema):
+    """Parser for show wireless fabric vnid mapping"""
+
+    cli_command = 'show wireless fabric vnid mapping'
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command)
+        else:
+          output = output
+
+        # Fabric VNID Mapping:
+        #   Name               L2-VNID        L3-VNID        IP Address             Subnet       Control plane name
+        # ----------------------------------------------------------------------------------------------------------------------
+        #   Data                8190           0                                  0.0.0.0            default-control-plane
+        #   Guest               8189           0                                  0.0.0.0            default-control-plane
+        #   Voice               8191           0                                  0.0.0.0            default-control-plane
+        #   Fabric_A_INF_VN     8188           4097           10.8.132.0          255.255.254.0      default-control-plane
+        #   Physical_Security     8192           0                                  0.0.0.0            default-control-plane
+
+        # Fabric VNID Mapping:
+        p_fabric_mapping = re.compile(r"^Fabric\s+VNID\s+Mapping:")
+
+        # Name               L2-VNID        L3-VNID        IP Address             Subnet       Control plane name
+        p_fabric_header = re.compile(r"^Name\s+L2-VNID\s+L3-VNID\s+IP\s+Address\s+Subnet\s+Control\s+plane\s+name$")
+
+        # ----------------------------------------------------------------------------------------------------------------------
+        p_fabric_delmimter = re.compile(r"^----------------------------------------------------------------------------------------------------------------------$")
```

Comment:
Probably not necessary to have these checks if you're continuing if they're matched

Reply:
Removed those regex matches